curations/crate/cratesio/-/wasi.yaml

```diff
@@ -3,6 +3,24 @@ coordinates:
   provider: cratesio
   type: crate
 revisions:
+  0.14.2+wasi-0.2.4:
```

Comment:
There is no license here?

Reply:
There is a license file present, however I am just following the pattern which is being followed in this file already. I don't see the option to declare license here.